c7n/resources/eks.py

```diff
@@ -211,6 +211,72 @@ def process(self, resources):
                 "Filtered %d of %d clusters due to state", state_filtered, len(resources))
 
 
+@EKS.action_registry.register('associate-encryption-config')
+class AssociateEncryptionConfig(Action):
+    schema = {
```

Comment:
please add a doc string with an example policy, that allows this to get into the references docs on the action.